qute.jdt/com.redhat.qute.jdt.test/src/main/java/com/redhat/qute/jdt/template/TemplateGetDataModelProjectTest.java

```diff
@@ -212,24 +215,64 @@ private static void testValueResolvers(DataModelProject<DataModelTemplate<DataMo
 				"fmtInstance(format : java.lang.String, ignoredPropertyName : java.lang.String, args : java.lang.Object[]) : java.lang.String",
 				"io.quarkus.qute.runtime.extensions.StringTemplateExtensions", resolvers);
 		assertValueResolver("str",
-				"str:fmt(ignoredPropertyName : java.lang.String, format : java.lang.String, args : java.lang.Object[]) : java.lang.String",
+				"fmt(ignoredPropertyName : java.lang.String, format : java.lang.String, args : java.lang.Object[]) : java.lang.String",
 				"io.quarkus.qute.runtime.extensions.StringTemplateExtensions", resolvers);
 
 		// from io.quarkus.qute.runtime.extensions.TimeTemplateExtensions
 		assertValueResolver(null,
 				"format(temporal : java.time.temporal.TemporalAccessor, pattern : java.lang.String) : java.lang.String",
 				"io.quarkus.qute.runtime.extensions.TimeTemplateExtensions", resolvers);
 		assertValueResolver("time",
-				"time:format(dateTimeObject : java.lang.Object, pattern : java.lang.String) : java.lang.String",
+				"format(dateTimeObject : java.lang.Object, pattern : java.lang.String) : java.lang.String",
 				"io.quarkus.qute.runtime.extensions.TimeTemplateExtensions", resolvers);
 
 	}
 
+	private void testValueResolversFomInject(List<ValueResolverInfo> resolvers) {
+		Assert.assertNotNull(resolvers);
+		Assert.assertFalse(resolvers.isEmpty());
+
+		// from org.acme.qute.InjectedData source
+
+		// @Named
+		// private String foo;
+		assertValueResolver("inject", "foo : java.lang.String", "org.acme.qute.InjectedData", //
+				null, resolvers);
+
+		// @Named("bar")
+		// private String aBar;
+		assertValueResolver("inject", "aBar : java.lang.String", "org.acme.qute.InjectedData", //
+				"bar", resolvers);
+
+		// @Named("user")
+		// private String getUser() {...
+		assertValueResolver("inject", "getUser() : java.lang.String", "org.acme.qute.InjectedData", //
+				"user", resolvers);
+
+		// @Named
+		// private String getSystemUser() {...
+		assertValueResolver("inject", "getSystemUser() : java.lang.String", "org.acme.qute.InjectedData", //

```

Comment:
I don't think that's a valid use case. CDI would expose a bean named getSystemUser or maybe systemUser (but I'm not that familiar with CDI).
I suggest you try running an actual Quarkus app with this @Named beans to check whether it makes sense or not.